Alexey_Michaylov/public_html/containers/BlogpostListContainer.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { PureComponent, Fragment } from 'react';
+import BlogpostList from 'components/BlogpostList';
+
+export default class BlogpostListContainer extends PureComponent {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            next: false,
+            prev: true,
+            posts: [],
+            page: 0,
+            countPage: 10,
```

Comment:
```suggestion
            countPage: 10,
```
Знаю, что лучше сходить и посчитать количество страниц)